src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -230,7 +233,9 @@ tag_invoke(boost::json::value_from_tag, boost::json::value& jv, LedgerEntryHandl
         {JS(validated), output.validated},
         {JS(index), output.index},
     };
-    object["deleted_ledger_index"] = output.deleted_ledger_index.has_value()? output.deleted_ledger_index.value() : 0;
+    if (output.deleted_ledger_index.has_value()) {
+        object["deleted_ledger_index"] = output.deleted_ledger_index.value();
+    }
```

Comment:
this would work, but i think we should use the optional pointer syntax just to be consistent with `nodeBinary`